holoviews/plotting/mpl/plot.py

```diff
@@ -262,9 +262,6 @@ class GridPlot(CompositePlot):
     show_legend = param.Boolean(default=False, doc="""
         Legends add to much clutter in a grid and are disabled by default.""")
 
-    tick_format = param.String(default="%.2f", doc="""
```

Comment:
Hmm, yes, we have a system for tick formatting using ``Dimension.value_format`` and ``Dimension.type_formatters``, so this is redundant. You think we need to go through a deprecation cycle for this?